decidim-proposals/spec/lib/decidim/proposals/markdown_to_proposals_spec.rb

```diff
@@ -202,7 +202,7 @@ def proposal_should_conform(section_level, title, body)
 
           it "are articles" do
             should_parse_and_produce_proposals(1)
-            proposal_should_conform(:article, "1", list)
+            expect { proposal_should_conform(:article, "1", list) }.not_to raise_error
```

Comment:
Same here, let's just inline the method

Reply:
fixed by [817b795](https://github.com/decidim/decidim/pull/14508/commits/817b7951b7bfcb78e1077477d00efc91d32f42f1)